lib/modules/hosts/strawpoll.js

```diff
@@ -0,0 +1,11 @@
+addLibrary('mediaHosts', 'strawpoll', {
+	domains: ['strawpoll.me'],
+	detect: href => (/^https?:\/\/(?:www\.)?strawpoll\.me\/(?:embed_[0-9]\/)?([0-9]*)/i).exec(href),
+	handleLink(elem, [, uid]) {
+		elem.type = 'IFRAME';
```

Comment:
I think it would make sense to add `elem.expandoClass = 'selftext';` here, so it uses the text expando icon